### ISSUE TITLE
Formatting for `$PSStyle.Formatting` labels `Verbose` as `Formatting.Verbose`

### ISSUE THREAD

**Issue by SeeminglyScience** [PowerShell/PowerShell#15907]
### Steps to reproduce

```powershell
$PSStyle.Formatting
```

### Expected behavior

```console
FormatAccent : `e[32;1m
TableHeader  : `e[32;1m
ErrorAccent  : `e[36;1m
Error        : `e[31;1m
Warning      : `e[33;1m
Verbose      : `e[33;1m
Debug        : `e[33;1m
```

### Actual behavior

```console
FormatAccent       : `e[32;1m
TableHeader        : `e[32;1m
ErrorAccent        : `e[36;1m
Error              : `e[31;1m
Warning            : `e[33;1m
Formatting.Verbose : `e[33;1m
Debug              : `e[33;1m
```

### Environment data

```powershell
Name                           Value
----                           -----
PSVersion                      7.2.0-preview.8
PSEdition                      Core
GitCommitId                    7.2.0-preview.8
OS                             Microsoft Windows 10.0.18363
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by iSazonov** [comment]
Fixed in #15825?

**Comment by SeeminglyScience** [comment]
It sure is. Good catch, thanks!
